package.json

```diff
@@ -61,10 +65,19 @@
     "eslint": "^3.3.1",
     "eslint-config-babel": "^1.0.1",
     "eslint-plugin-babel": "^3.3.0",
-    "eslint-plugin-flow-vars": "^0.5.0"
+    "eslint-plugin-flow-vars": "^0.5.0",
+    "mocha": "^3.0.2"
   },
   "babel": {
     "plugins": [
+      "babel-plugin-transform-es2015-destructuring",
```

Comment:
Since babel supports node 0.10 still maybe we should be using the same transform as that (although we want to drop it in babel 7)? We can always make this 4.x only as well